stories/Image.stories.mdx

```diff
@@ -280,7 +321,12 @@ import { Image } from '@wpmedia/engine-theme-sdk';
       mediumHeight={154}
       largeWidth={274}
       largeHeight={154}
-      resizedImageOptions={{}}
```

Comment:
Same with this, this story is to show what breaks. 

A couple of the example images do not load – is that intentional? Does this component not work without these params – if so, should we say in the documentation that they’re required? 

Without resizer URL 

No resized params

Reply:
I do say that

Reply:
Its been added back